src/super_gradients/recipes/imagenet_repvgg.yaml

```diff
@@ -4,20 +4,21 @@
 #  Reach => 72.05 Top1 accuracy.
 #
 #  Log and tensorboard at s3://deci-pretrained-models/repvggg-a0-imagenet-tensorboard/
-
+#
```

Comment:
Don't think this file was supposed to be in this PR...

Reply:
This was part of previous PR that is now merged. Now it is removed fro this PR.